aszczachor/src/components/navigation/ExpandButton.js

```diff
@@ -0,0 +1,29 @@
+import expandArrow from "assets/mobileMenu/expand-arrow.png";
+import collapseArrow from "assets/mobileMenu/collapse-arrow.png";
+import { Link } from "react-router-dom";
+
+const ExpandButton = ({ label, changeState, isOpen, list, openMenu }) => {
+  return (
+    <>
+      <li key={label} style={{ marginLeft: "17px" }}>
+        <button onClick={changeState}>
+          {label}
+          <img src={isOpen ? collapseArrow : expandArrow} alt="" />
+        </button>
+      </li>
+      {isOpen
+        ? list.map(item => {
```

Comment:
nie wiem czy zamierzone czy nie ale możesz to item destrukturyzować na ({path, label})
PS. w wielu miejscach to widzę, jeżeli jest dość prosta domean to warto destrukturyzować, jeżeli cieżko skminić o co chodzi, czasami warto zostawić tak jak masz